params/params.go

```diff
@@ -242,6 +243,9 @@ type Params struct {
 	LedgerParams ledger.LedgerParams
 
 	CoinbaseConfig CoinbaseConfigs
+
+	// min gbt duration when empty transactions
+	MinEmptyGbtDuration time.Duration
```

Comment:
Will its purpose feel like it will affect `TargetTimePerBlock,` and will it make HashRate's calculation unrealistic